ref/src/yppm_driver.f90

```diff
@@ -6,25 +6,11 @@
 program yppm_driver
 
   use OMP_LIB
-  use yppm_core_mod, ONLY: yppm
+  use yppm_core_mod
   use netCDFModule
```

Comment:
I think you can probably get rid of this, too?

Reply:
@christopherwharrop-noaa Sure can! This slipped through the cracks.